### ISSUE TITLE
Why 3 training loss always

### ISSUE THREAD

**Issue by nazib** [balakg/voxelmorph#2]
I was training your network with my own dataset. While training it always generates 3 training losses as tuple in 'train_loss' variable. But the network is setup with two loss function. Checking the loss values i found that two losses are exactly same which I guess cross-correlation loss and the third one is the regularisation loss. Is that correct? 

However thanks for this great code. really helpful

**Comment by balakg** [comment]
The first loss is the total loss (weighted sum of both losses), and the next two are the cross-correlation and reg. losses. Keras happens to always output the total loss first.

**Comment by nazib** [comment]
Thanks
